packages/web-components/src/global/variables.css

```diff
@@ -1,40 +1,102 @@
-/* Colors from: */
-/*   https://design.va.gov/design/color-palette */
-/* Fonts from: */
-/*   https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/8d39616e957cb44b6664a658b2c51dca8076fe7e/packages/formation/sass/base/_b-variables.scss#L16-L17 */
-:root {
+/**
+ * Do not edit directly
+ * Generated on Wed, 26 Jul 2023 21:34:13 GMT
+ */
+
+ :root {
   --color-base: #212121;
-  --color-black: #000000;
   --color-white: #ffffff;
+  --color-black: #000000;
+  --color-orange: #eb7f29;
+  --color-link-default: #004795;
+  --color-warning-message: #fac922;
+  --color-gibill-accent: #fff1d2;
+  --color-primary: #0071bb;
+  --color-primary-darker: #003e73;
+  --color-primary-darkest: #112e51;
+  --color-primary-alt: #02bfe7;
+  --color-primary-alt-dark: #00a6d2;
+  --color-primary-alt-darkest: #046b99;
+  --color-primary-alt-light: #9bdaf1;
+  --color-primary-alt-lightest: #e1f3f8;
+  --color-secondary: #e31c3d;
+  --color-secondary-dark: #cd2026;
+  --color-secondary-darkest: #981b1e;
+  --color-secondary-light: #e59393;
+  --color-secondary-lightest: #f9dede;
   --color-gray: #5b616b;
-  --color-gray-medium: #757575;
   --color-gray-dark: #323a45;
+  --color-gray-medium: #757575;
   --color-gray-light: #aeb0b5;
+  --color-gray-light-alt: #eeeeee;
   --color-gray-lighter: #d6d7d9;
   --color-gray-lightest: #f1f1f1;
+  --color-gray-warm-dark: #494440;
+  --color-gray-warm-light: #e4e2e0;
   --color-gray-cool-light: #dce4ef;
   --color-green: #2e8540;
+  --color-green-dark: #1e5f2f;
+  --color-green-darker: #195c27;
+  --color-green-darkest: #0f3f1f;
+  --color-green-light: #7dbf7c;
   --color-green-lighter: #94bfa2;
   --color-green-lightest: #e7f4e4;
   --color-gold: #fdb81e;
   --color-gold-light: #f9c642;
   --color-gold-lighter: #fad980;
   --color-gold-lightest: #fff1d2;
-  --color-link-default: #004795;
-  --color-link-visited: #4c2c92;
-  --color-primary: #0071bb;
-  --color-primary-darker: #003e73;
-  --color-primary-darkest: #112E51;
-  --color-primary-alt-dark: #00a6d2;
-  --color-primary-alt-darkest: #046b99;
-  --color-primary-alt-light: #9bdaf1;
-  --color-primary-alt-lightest: #e1f3f8;
-  --color-secondary-dark: #cd2026;
-  --color-secondary-light: #e59393;
-  --color-secondary-lightest: #f9dede;
   --v3-color-error-dark: #b50909;
   --v3-color-base-darkest: #1B1B1B;
-  --font-source-sans: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Roboto,
-    Arial, sans-serif;
-  --font-serif: Bitter, Georgia, Cambria, 'Times New Roman', Times, serif;
```

Comment:
I'm doing a spot check for these modified tokens just in case something got accidentally removed. Most of them just seem to be in a different order but there are a couple that I think got removed. Can you double check them and make sure any that aren't already in the CSS Library tokens files get added? Here are a few I found that were removed:

```
--font-source-sans
--font-serif
--color-link-visited
```

Reply:
These should be re-added now. I checked the other removed lines and they all seem to be re-added elsewhere in the file.